token/sdk/tms/db/tms.go

```diff
@@ -157,3 +191,20 @@ func (p *PostInitializer) ConnectNetwork(networkID, channel, namespace string) e
 	}
 	return nil
 }
+
+type TransactionFilter struct {
```

Comment:
This one could be private I think. Also, maybe it would make sense to give it a name that describes also what it does, e.g. discardUnknownTxFilter or unknownTxFilter.

Reply:
👍